Server/App/api/deck.py

```diff
@@ -181,7 +181,14 @@ def post(self, name, public, tags, cards, device, user_id, connector, uuid):
                     card_obj = CardPolicy.belongs_to(card['uuid'], deck.uuid, connector)
                     if card_obj is None:
                         raise ValueError  # not allowed to edit this card
-                    card_obj.edit(deck.uuid, card['front'], card['back'], card['position'])
+
+                    # Make edits.
+                    if card.get('front') is not None or card.get('back') is not None:
+                        card_obj.edit(deck.uuid, card.get('front'), card.get('back'))
+
+                    # Move card.
+                    if card.get('position') is not None:
+                        card_obj.move(deck.uuid, card['position'])
```

Comment:
I sort of feel like that "front" "back" "end" should be expecting? Since this endpoint is supposed to be for overwriting